src/styles.css

```diff
@@ -232,49 +238,53 @@ select {
 
 .pSmall {
   text-align: center;
-  font-size: 14px;
+  font-size: 1.4rem;
 }
 
 .purpleSubheading {
   color: rgb(143, 64, 156);
-  font-size: 22px;
-  margin: 20px 0px;
+  font-size: 2.2rem;
+  margin: 2rem 0;
   font-family: sans-serif;
 }
 
+.formText {
+  padding-bottom: 1rem;
+}
+
 .inputLabel {
   font-weight: bold;
-  line-height: 28px;
   font-family: sans-serif;
+  display: block;
 }
 
 .labelDetail {
-  margin: 0;
-  font-size: 14px;
+  font-size: 1.4rem;
 }
 
 .helpIcon {
-  height: 38px;
+  height: 3.8rem;
 }
 
-.textBoxHalfWidth,
 .textBox {
-  border: 1px solid rgb(160, 160, 160);
-  height: 32px;
-  font-size: 16px;
-  width: 60px;
-  padding: 4px;
-  margin-bottom: 10px;
+  border: 0.1rem solid rgb(160, 160, 160);

```

Comment:
You should only use rem for properties that you would like to be responsive. You wouldn't want your border width to change with screen size, so use px for border width